### PR TITLE
chore: formatting pass

### PR DESCRIPTION
**Pull request by fernando-cortez**
### Description
A formatting pass over the codebase to correct whitespace/styling issues. This is in anticipation of subsequent formatting CI jobs.

### Issue Number(s)
No ticket for this chore.

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] JIRA ticket ID is in the PR title or at least one commit message
 - [ ] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

**Review by LPLafontaineB — APPROVED** [review]

File: Assets/BossRoom/Scripts/Game/Client/UI/UnityServicesUIHandler.cs [hunk 1]
```diff
@@ -28,22 +28,22 @@ void ServiceErrorHandler(UnityServiceErrorMessage error)
             switch (error.AffectedService)
             {
                 case UnityServiceErrorMessage.Service.Lobby:
-                {
-                    HandleLobbyError(error);
-                    break;
-                }
+                    {
```
Comment (SamuelBellomo) [inline Assets/BossRoom/Scripts/Game/Client/UI/UnityServicesUIHandler.cs:31]: really? is that in our unity guidelines? That's so ugly!

Comment (fernando-cortez) [reply Assets/BossRoom/Scripts/Game/Client/UI/UnityServicesUIHandler.cs:31]: Really? I've always seen code inside a case be indented. Though I can't say the same for braces inside cases. Here, it's just applying the indentation rule.

**Review by pdeschain — APPROVED** [review]

File: Assets/BossRoom/Scripts/Game/Client/UI/UnityServicesUIHandler.cs (hunk 1 above)
Comment (SamuelBellomo) [reply Assets/BossRoom/Scripts/Game/Client/UI/UnityServicesUIHandler.cs:31]: According to our coding convention, braces must be aligned with the “case”
https://drive.google.com/file/d/1JT7jviC31yevd6PnkZUS-amkUFVOoGIQ/view
There’s even an example for it in there.

Comment (fernando-cortez) [reply Assets/BossRoom/Scripts/Game/Client/UI/UnityServicesUIHandler.cs]: dotnet format seems to still be adding the indent, even when modifying the .editorconfig file to explicitly not do this. Will circle back to this for now, but in the meantime, I've removed the braces.

**Review by SamuelBellomo — APPROVED** [review]

**Review by LPLafontaineB — APPROVED** [review]

**Pull request merged**